packages/client/test/miner/miner.spec.ts

```diff
@@ -259,8 +265,8 @@ describe('[Miner]', async () => {
     // no skipHardForkValidation
     const miner = new Miner({ config: goerliConfig, service })
     const { txPool } = service
+    await service.execution.setupMerkleVM()
```

Comment:
There are several (most except for 1) cases where this also works with `await service.execution.open()`, just tested, will replace since we should rather use the "public" (internal) API for things like that for setup which then closer resembles the real situation.